docs/datamap/lucene-datamap-guide.md

```diff
@@ -0,0 +1,180 @@
+# CarbonData Lucene DataMap
+  
+* [Quick Example](#quick-example)
+* [DataMap Management](#datamap-management)
+* [Lucene Datamap](#lucene-datamap-introduction)
+* [Loading Data](#loading-data)
+* [Querying Data](#querying-data)
+* [Data Management](#data-management-with-pre-aggregate-tables)
+
+## Quick example
```

Comment:
The below is a procedure, so put it in a numbered list: 
Step 1:
Step 2:

Reply:
added numbering for steps and sub headings are kept as it is